Startup fails

Startup fails with exception  

Cannot find module 'C:\MediacenterJS\node_modules\sqlite3\lib\binding\node-v46-win32-x64\node_sqlite3.node'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\MediacenterJS\node_modules\sqlite3\lib\sqlite3.js:4:15)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\MediacenterJS\lib\utils\database-schema.js:2:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)

I have tried installing sqlite3 globally...

Nice application btw, screenshots look really good.

I have a problem starting the application as well:

Here's the output:
```
npm WARN mediacenterjs@0.1.3 license should be a valid SPDX license expression
npm ERR! Linux 4.4.0-31-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE

npm ERR! sqlite3@3.0.5 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sqlite3@3.0.5 install script 'node-pre-gyp install --fallback-to-build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the sqlite3 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-pre-gyp install --fallback-to-build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs sqlite3
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls sqlite3
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/osboxes/mediacenterjs/npm-debug.log

```

--- a/package.json
+++ b/package.json
@@ -52,7 +52,7 @@
         "semver": ">=2.0.0",
         "sequelize": "^3.0.0",
         "socket.io": "^0.9.16",
-        "sqlite3": "3.0.5",
+        "sqlite3": "~3.1.4",
         "to-title-case": "^0.1.5",
         "umzug": "^1.6.0",
         "underscore": "^1.7.0",

